vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -119,6 +119,28 @@ instances:
     #   cluster:
     #     - <CLUSTER_REGEX>            # Only possible with "collection_type: historical"
 
+    ## @param metric_instance_values_filters - object - optional - default: none
+    ## For each resource type (vm, host, datastore, datacenter, cluster) to collect,
+    ## you can choose which metric you want to collect the instance value using a list of regex.
+    ## See https://github.com/DataDog/integrations-core/blob/master/vsphere/datadog_checks/vsphere/metrics.py
+    ## for the list of collected metrics (do not prefix then with `vsphere`)
+    ## Caveat: for a given metric if we collect the instance value (matched), but the instance value is empty,
+    ## the metric is skipped. This is to avoid submitting aggregated metrics (empty instance value)
+    ## when we already submit the metrics tagged by the instance value. Hence the regex used should be precise enough
+    ## to avoid skipping metrics that does not have instance values.
+    #
+    # metric_instance_values_filters:
```

Comment:
Wondering if we can find a better name `collect_per_instance_filters` maybe? Not 100% convinced either but keeping the discussion open on this

Reply:
`collect_per_instance_filters` sounds good and more consistent since we are using mostly the term `per instance` elsewhere.